WAW.API/Chat/Domain/Models/Message.cs

```diff
@@ -0,0 +1,16 @@
+using WAW.API.Auth.Domain.Models;
+using WAW.API.Shared.Domain.Model;
+
+namespace WAW.API.Chat.Domain.Models;
+
+public class Message : BaseModel {
+  public string MessageContent { get; set; } = string.Empty;
+  public DateTime Date { get; set; }
+
+  //Relationships
+  public long UserId { get; set; }
+  public User User { get; set; }
```

Comment:
Maybe this would be better:

```suggestion
  public long SenderId { get; set; }
  public User Sender { get; set; }
```